corehq/apps/cleanup/management/commands/repeater_times.py

```diff
@@ -13,7 +12,7 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         domain = args[0]
-        db = Domain.get_db()
+        db = RepeatRecord.get_db()
```

Comment:
`receiverwrapper` has had its own DB for a long time now, so this managemtn command can't have worked for a long time. Given that, I'd opt to just delete it!